Doc/library/traceback.rst

```diff
@@ -8,11 +8,12 @@
 
 --------------
 
-This module provides a standard interface to extract, format and print stack
-traces of Python programs.  It exactly mimics the behavior of the Python
-interpreter when it prints a stack trace.  This is useful when you want to print
-stack traces under program control, such as in a "wrapper" around the
-interpreter.
+This module provides a standard interface to extract, format and print
+stack traces of Python programs.  It can mimic the behavior of the Python
+interpreter when it prints a stack trace, which is useful when you want
+to print stack traces under program control. It is also more flexible
```

Comment:
I don't feel like I understand what "under program control" means here, though that's admittedly a pre-existing issue with the docs here

Reply:
I agree. I think this whole sentence can be removed.